backend/tests/services/object_storage/test_presign_service.py

```
@@ -0,0 +1,35 @@
+from minio import Minio
+
+from app.services.object_storage.presign_service import ObjectStoragePresignService
+
+
+def test_generate_download_url_supports_long_lived_expiration(monkeypatch):
+    """Generate publish download URLs longer than the MinIO client default limit."""
+    service = ObjectStoragePresignService()
+    client = Minio(
+        "s3.example.com",
+        access_key="access-key",
+        secret_key="secret-key",
```

Ruff (0.15.17)

[error] Possible hardcoded password assigned to argument: "secret_key"

(S106)